game/resource/English/ability/items/tooltip_meteor_hammer.txt

```diff
@@ -1,3 +1,11 @@
+"DOTA_Tooltip_Ability_item_meteor_hammer_1"                        "Meteor Hammer"
+"DOTA_Tooltip_ability_item_recipe_meteor_hammer_1"                 "Recipe: Meteor Hammer"
+"DOTA_Tooltip_ability_item_meteor_hammer_1_Description"                   "<h1>Active: Meteor Hammer</h1> CHANNELED - After a succesful channel, summons a meteor that strikes a %impact_radius% AoE, stunning enemies for %stun_duration% seconds and dealing impact damage. Continues to deal damage over time to enemies units and buildings for %burn_duration% seconds.<BR><BR>Building Impact Damage: %impact_damage_buildings% <BR>Building Over Time Damage: %burn_dps_buildings% <BR><BR>Non-Building Impact Damage: %impact_damage_units% <BR>Non-Building Over Time Damage: %burn_dps_units% <BR><BR>Channel Duration: %max_duration% seconds.<BR>Landing Time: %land_time% seconds."
```

Comment:
You forgot to change the tooltip to not talk about buildings. It's also referencing KVs that don't exist since you changed the key names.